app/lib/linters/crunch_api.rb

```diff
@@ -5,34 +5,26 @@ module Linters
   # A class called Crunch
   class CrunchApi
 
-    attr_accessor :base_url
-
-    def initialize()
-      @base_url = "https://platform.42crunch.com"
+    def initialize(file:, base_url: "https://platform.42crunch.com")
```

Comment:
This base URL probably wants to go in the .env file

Reply:
Done